src/games/strategy/engine/framework/mapDownload/DownloadMapDialog.java

```diff
@@ -1,207 +0,0 @@
-package games.strategy.engine.framework.mapDownload;
```

Comment:
DownloadMapDialog.java - this was the interstitial map download dialog that popped up asking the user which URL they wanted to use to get the 'triplea_maps.xml' file